### PR TITLE
Fixes #1823 - Replace FLASK_ENV with FLASK_DEBUG

### LINKED ISSUE #1823: FLASK_ENV is deprecated
`FLASK_ENV` has been deprecated since 2.2.0: https://flask.palletsprojects.com/en/2.2.x/changes/?highlight=flask_env#version-2-2-0

This can be fixed with setting `FLASK_DEBUG` instead of `FLASK_ENV`: https://flask.palletsprojects.com/en/2.2.x/config/#DEBUG

This should be fixed to remove deprecation warning:
```ShellSession
$ flask run --debugger --with-threads --reload --port 5000
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
```

### PR DESCRIPTION
**Pull request by sixcare**
This PR aims to remove deprecation warning for FLASK_ENV, solving #1823

### CODE REVIEW

File: alerta/utils/logging.py [hunk 1]
```diff
@@ -214,8 +214,8 @@ def format(self, record):
         }
         payload['message'] = record.getMessage()
 
-        # do not assume there's a Flask request context here so must use FLASK_ENV env var not app.debug
-        indent = 2 if os.environ.get('FLASK_ENV', '') == 'development' else None
+        # do not assume there's a Flask request context here so must use FLASK_DEBUG env var not app.debug
+        indent = 2 if os.environ.get('FLASK_DEBUG', False) else None
```
Comment (satterly) [inline alerta/utils/logging.py:218]: FLASK_DEBUG=1 to set debug ...
```
indent = 2 if os.environ.get('FLASK_DEBUG', '') == '1' else None
```

File: README.md [hunk 2]
```diff
@@ -76,14 +76,14 @@ Development
...
-    $ export FLASK_APP=alerta FLASK_ENV=development
+    $ export FLASK_APP=alerta FLASK_DEBUG=True
     $ pip install -e .
     $ flask run
 
 To run in development mode, listening on port 8080, using Postgres and
 reporting errors to [Sentry](https://sentry.io):
 
-    $ export FLASK_APP=alerta FLASK_ENV=development
+    $ export FLASK_APP=alerta FLASK_DEBUG=True
```
Comment (satterly) [inline README.md:86]: FLASK_DEBUG=1

File: README.md [hunk 3]
```diff
@@ -76,14 +76,14 @@ Development
 
 To run in development mode, listening on port 5000:
 
-    $ export FLASK_APP=alerta FLASK_ENV=development
+    $ export FLASK_APP=alerta FLASK_DEBUG=True
```
Comment (satterly) [inline README.md:79]: FLASK_DEBUG=1

File: .flaskenv [hunk 4]
```diff
@@ -1,2 +1,2 @@
-FLASK_ENV=development
+FLASK_DEBUG=True
```
Comment (satterly) [inline .flaskenv:1]: FLASK_DEBUG=1

**Review by satterly — CHANGES_REQUESTED** [review]

File: .flaskenv (hunk 4 above)
Comment (satterly) [reply .flaskenv:1]: See https://github.com/pallets/flask/issues/4714 PR description for usage.

Comment (sixcare) [reply .flaskenv:1]: Both work: https://github.com/pallets/flask/blob/main/src/flask/helpers.py#L28 but I can of course set to an int

Comment (sixcare) [reply .flaskenv]: Done https://github.com/alerta/alerta/pull/1824/commits/1f74ee217e2eab282afa4941bc095ae348895106 :)

**Review by satterly — APPROVED** [review]

**Comment by satterly** [comment]
Thanks!

**Pull request merged**